.travis.yml

```diff
@@ -5,7 +5,8 @@ php:
   - 7.2
 
 env:
-   - SYMFONY_VERSION=4.0.*
+  - SYMFONY_VERSION=3.4.0
```

Comment:
Why not try all `3.4.*` versions ?

Reply:
because we say in composer.json that we support `3.4.0`, so we have to build with this specific minimal version to be sure :)